packages/core/src/layout/FlexLayout/FlexLayout.css

```diff
@@ -23,40 +23,50 @@
   align-items: var(--flex-layout-align);
 }
 
-.uitkFlexLayout-separator {
-  --flex-layout-gap: calc(var(--uitk-size-unit) * var(--flex-layout-gap-multiplier) * 2 + var(--flex-layout-separator));
+.uitkFlexLayout-separator-wrapper {
+  display: contents;
```

Comment:
`display: contents` could potentially be an accessibility issue on Safari (see [caniuse](https://caniuse.com/?search=display%20contents)). Having said that I think it's used on the uitk-theme element which could be a bigger issue @heswell what do you think?

Reply:
from what i understand, it is an accessibility issue that would arise from adding accessibility props directly into the wrapper, in this case the component is adding the wrapper after mapping children, so if you wanted to add for example a role to your children, those would still apply since it is only the outside wrapper that would have `display: contents` (but I'll add a test for Safari to make sure I'm understanding this correctly)

Reply:
Oh cool I had thought it applied to children of the `display: contents` element but I could well be wrong, in which case it should be fine to use :)

Reply:
I've come to think that the uitk-theme element was a mistake and that the default behaviour should be to add classes to nested child, with the custom element as the configurable option . And yes, it does use display: content , which kind of does the trick but also has the potential to break users css rules if they are not expecting this element to be present.